Terran-One/cw-simulate
# Bank Module

Implementation of bank module w/ unit & integration tests

Comes with a refactor for more convenient interaction with our `cw-simulate-tests` contract (i.e. includes changes to wasm.spec.ts). Could need some more tests, e.g. failure coverage

## Test steps
Run `npm test src/modules/bank.spec.ts` and confirm no tests fail (tho we have todos).
